config/fxdata/trapdoor.cfg

```diff
@@ -548,3 +548,84 @@ SellingValue = 1350
 Unsellable = 1
 PlaceSound = 117
 TriggerSound = 141
+
+[trap8]
+Name = BALISTA
+NameTextID = 814
+TooltipTextID = 814
+SymbolSprites = BALISTA_BIG BALISTA_SMALL
+PointerSprites = BALISTA_CURSOR
+PanelTabIndex = 15
+Crate = WRKBOX_BALISTA
+ManufactureLevel = 3
+ManufactureRequired = 20000
+Shots = 255
+ShotOrigin = 0 50 160
+TimeBetweenShots = 80
+SellingValue = 1200
+AnimationID = BALISTA_STAND
+AttackAnimationID = BALISTA_ATTACK
+RechargeAnimationID = BALISTA_RELOAD_LONG_ANIM
+ModelSize = 220
+AnimationSpeed = 0
+AttackAnimationSpeed = 500
+RechargeAnimationSpeed = 62
+TriggerType = 3
+ActivationType = 5
+EffectType = SHOT_BALISTA
+Hidden = 0
+Destructible = 1
+PlaceOnBridge = 0
+Health = 1000
+Unshaded = 0
+RandomStartFrame = 0
+ThingSize = 256 320
+HitType = 4
+LightRadius = 6
+LightIntensity = 32
+LightFlags = 0
+TransparencyFlags = 0
+ShotVector = 0 0 0
+PlaceSound = 117
+RechargeSound = 62
```

Comment:
Yeah, you want the recharge sound for sure? If so #4218 needs to be merged before.
Also, not wanting a different sound from the internet?